### DISCUSSION TITLE
More compact and configurable Live QSO page

### DISCUSSION THREAD

**Discussion by filipmelik** [wavelog/wavelog#1282] [Ideas]
Hi! 

1) Sometimes I use Wavelog for logging random ad-hoc contacts via my mobile phone. But when you open Live QSO page on mobile, the whole page is unnecessary tall - I made quick and dirty mod on my instance, to make date&time + RST fields fit on the one line, which in turn saves 2 lines of precious space, and make me scroll a little less to reach the Save button :) 

Before:
<img width="342" alt="before" src="https://github.com/user-attachments/assets/7893af26-5ad5-4bc1-9966-d37be73cebb0">

After:
<img width="311" alt="after" src="https://github.com/user-attachments/assets/93a1544f-218b-4ecf-a4fc-ebdac21afedb">

So I was wondering if this feels useful and if I  should submit the PR with this change?

2) While doing this mod, I thought that I would maybe hide some fields I do not personally use. I have notices in the source code, that there already is plenty of fields that are toggleable. That led me to idea, if you possibly thought about whole QSO page fields   being able to be toggleable per user preference :)

Thx and 73!

**Comment by HB9HIL** [comment]
You don’t need permission to create PRs :-) You can always create one, but remember that there is no guarantee a PR will get merged.

PR's always against `dev` branch

**Reply by HB9HIL** [reply]
Some more Information can be found here https://github.com/wavelog/wavelog/wiki/Development-Notes

**Reply by filipmelik** [reply]
I know I do not need permission :)  The reason I am asking is whether this feature even makes sense to you before opening a PR :)

Anyway, what about the point 2 in my previous message? Have you considered something like "configurable" QSO page before? Or is this too much effort for small benefit as I suspect the code changes would not be that trivial? :)

**Reply by HB9HIL** [reply]
Tbh: We need to set a border where things are configurable. Unless you never stop.. So I don't see this atm

**Reply by filipmelik** [reply]
NP, and totally understood:) btw this is the reason I am asking those things first, before investing time in that 😊

**Reply by HB9HIL** [reply]
Small things like the current PR are no problem. Big changes is good to request beforehand. You're right with that.
